modules/nw-multus-host-device-object.adoc

```diff
@@ -1,30 +1,47 @@
 // Module included in the following assemblies:
 //
+// * networking/multiple-networks/configuring-host-device.adoc
 
 [id="nw-multus-host-device-object_{context}"]
 = Configuration for host-device
 
-// TODO - consider refactoring this into its own module as it is used
-// by both ipvlan and bridge CNI plug-ins.
+The configuration for an additional network attachment that uses the host-device
+Container Network Interface (CNI) plug-in is provided in two parts:
 
-.host-device CNI plug-in YAML configuration
+* Cluster Network Operator (CNO) configuration
+* CNI plug-in configuration
+
+The CNO configuration specifies the name for the additional network attachment
+and the Kubernetes namespace to create the attachment in. The plug-in
+is configured by a JSON object specified by the `rawCNIConfig` parameter in
+the CNO configuration.
+
+The following YAML describes the configuration parameters for the CNO:
+
+.Cluster Network Operator YAML configuration
 [source,yaml]
 ----
 name: <name> <1>
 namespace: <namespace> <2>
-rawCNIConfig: '' <3>
+rawCNIConfig: '{ <3>
+  ...  
+}'
 type: Raw
 ----
-<1> `name`: Specify the name of the `NetworkAttachmentDefinition` created from
-the `rawCNIConfig` JSON object.
+<1> Specify a name for the additional network attachment that you are
+creating. The name must be unique within the specified `namespace`.
+
+<2> Specify the namespace to create the network attachment in. If
+you do not specify a value, then the `default` namespace is used.
 
-<2> `namespace`: Specify the namespace to create the network attachment in. If
-a value is not specified, the `default` namespace is used.
+<3> Specify the CNI plug-in configuration in JSON format, which
+is based on the following template.
 
-<3> `rawCNIConfig`: Specify the CNI plug-in configuration.
+IMPORTANT: Specify your network device by setting only one of the
```

Comment:
Should you use the note format?
```
[IMPORTANT]
====
Specify your network device by setting only one of the following parameters: `device`, `hwaddr`, `kernelpath`, or `pciBusID`.
====
```

Reply:
@mburke5678, I thought so too, but I saw this syntax used elsewhere, and it renders correctly in the output. This is the single line version and it works for all note types.